src/Components/Voting.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import { Button } from "react-bootstrap";
+import { patchArticle } from "../utils/api";
+
+export default function Voting({ votes, article_id }) {
+  const [vote, setVote] = useState(0);
+  const [alerts, setAlerts] = useState(false);
+
+  const incVote = () => {
+    setVote((currChange) => {
+      patchArticle(article_id, 1);
+      return currChange + 1;
+    });
+  };
+
+  const decVote = () => {
+    setVote((currChange) => {
+      patchArticle(article_id, -1);
+
+      return currChange - 1;
+    });
+  };
+
```

Comment:
Make sure you handle unsuccessful patch requests here - if you are handling votes with an alert a catch block should change the alert text to tell the user this.